src-tauri/src/main.rs

```diff
@@ -16,12 +16,14 @@ mod manager;
 
 static HANDLE: OnceLock<Mutex<AppHandle>> = OnceLock::new();
 lazy_static! {
-    static ref SHARED_CONDVAR: (Mutex<bool>, Condvar) = (Mutex::new(false), Condvar::new());
+    static ref HANDLE_CONDVAR: (Mutex<bool>, Condvar) = (Mutex::new(false), Condvar::new());
+    static ref WATCHER_CONDVAR: (Mutex<bool>, Condvar) = (Mutex::new(false), Condvar::new());
+    static ref WATCHER_STATE: Mutex<Vec<&'static str>> = Mutex::new(Vec::new());
```

Comment:
I'd like a comment for each of these describing how they're used. I'm not sure that this is the best way to manage state.

Will ask ChatGPT.